htsget-http-actix/benches/request_benchmarks.rs

```diff
@@ -152,7 +154,7 @@ fn start_htsget_rs() -> (DropGuard, String) {
     .unwrap();
 
   let htsget_rs_url = format!("http://{}", config.addr);
-  query_server_until_response(format_url(&htsget_rs_url, "reads/service-info"));
+  query_server_until_response(format_url(&htsget_rs_url, "reads/service-info"), base_path.into_path());
```

Comment:
I think the problem is that the `query_server_until_response` function only checks for connection to the htsget query server (i.e. the 8080 port server), and not the ticket server (8081 port server). For absolute correctness, it should also wait on the ticket server. This could be done by adding something like the following to the `start_htsget_rs` function:

```Rust
  let htsget_rs_ticket_url = format!("http://{}", config.ticket_server_addr);
  query_server_until_response(format_url(&htsgett_rs_ticket_url, ""));
```

Reply:
Thanks Marko, added in your suggestions... Do you think that the DropGuard have to be around that last `htsget_rs_ticket_url` too?

Reply:
Nope.

The `DropGuard` is only used for the spawned process. `htsget_rs_ticket_url` is not needed after its use in `start_htsget_rs` - tickets are obtained using the responses in the `request` function anyway.